app/src/main/res/raw/auth_config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "client_id" : "Register your app at https://aka.ms/MobileAppReg",
+  "client_id" : "Enter_the_Application_Id_here",
   "authorization_user_agent" : "DEFAULT",
-  "redirect_uri" : "Register your app at https://aka.ms/MobileAppReg",
+  "redirect_uri" : "Enter_the_Redirect_URI_Here",
```

Comment:
I believe here you need to add 
`"account_mode: "MULTIPLE",
"broker_redirect_uri_registered": true,
`
To match what is in the config file on docs.microsoft. @rpdome can you please check this?